xml/ns-System.Drawing.xml

```diff
@@ -11,7 +11,7 @@
  The <xref:System.Drawing.Graphics> class provides methods for drawing to the display device. Classes such as <xref:System.Drawing.Rectangle> and <xref:System.Drawing.Point> encapsulate GDI+ primitives. The <xref:System.Drawing.Pen> class is used to draw lines and curves, while classes derived from the abstract class <xref:System.Drawing.Brush> are used to fill the interiors of shapes.
 
 > [!CAUTION]
-> Classes within the <xref:System.Drawing> namespace are not supported for use within a Windows or ASP.NET service. Attempting to use these classes from within one of these application types may produce unexpected problems, such as diminished service performance and run-time exceptions. For a supported alternative, see [Windows Imaging Components](/windows/desktop/wic/-wic-about-windows-imaging-codec).
+> Classes within the <xref:System.Drawing> namespace are not supported for use within a Windows, ASP.NET Core, or ASP.NET service. Attempting to use these classes from within one of these application types may produce unexpected problems, such as diminished service performance and run-time exceptions. For a supported alternative, see [Windows Imaging Components](/windows/desktop/wic/-wic-about-windows-imaging-codec).
```

Comment:
Why not just delete this caution block and combine the info into includes/drawing.md?

Reply:
That include is used here and in two other places, so would probably be good to update it with ASP.NET Core service.

https://github.com/dotnet/dotnet-api-docs/search?q=includes%2Fdrawing.md